pkg/acsengine/const.go

```diff
@@ -90,6 +102,12 @@ var KubeImages = map[api.OrchestratorVersion]map[string]string{
 		"dnsmasq":      "k8s-dns-dnsmasq-amd64:1.13.0",
 		"pause":        "pause-amd64:3.0",
 		"windowszip":   "v1.6.0intwinnat.zip",
+		"nodestatusfreq":   "10s",
+		"nodegraceperiod":   "40s",
+		"podeviction":   "5m0s",
+		"routeperiod":   "10s",
```

Comment:
@jackfrancis @JackQuincy Hey Jacks! I'd like this feature to only be available on v1.6 k8s. It should not be toggleable on v1.5.7. Doable?